src/Admin/Pool.php

```diff
@@ -270,12 +271,12 @@ public function getAdminByAdminCode($adminCode)
 
             if (!$admin->hasChild($code)) {
                 @trigger_error(sprintf(
-                    'Passing an invalid admin hierarchy inside argument 1 for %s() is deprecated since sonata-project/admin-bundle 3.x and will throw an exception in 4.0.',
+                    'Calling "%s()" with an invalid admin hierarchy is deprecated since sonata-project/admin-bundle 3.x and will throw an exception in 4.0.',
                     __METHOD__
```

Comment:
AFAIK, the current form is widely adopted, see https://www.php.net/manual/es/migration73.deprecated.php#migration73.deprecated.core.string-search by instance.
Why do you consider this change required?

Reply:
It didn't make sense to me to explicitly name the position of the argument. If arguments would change, you would need to change these messages also... However, I changed it back.